### DISCUSSION TITLE
Update on the release timeline for Milvus version 2.5.0?

### DISCUSSION THREAD

**Discussion by villdie18** [milvus-io/milvus#37461] [Q&A and General discussion]
Hello Milvus team,

I wanted to ask about the expected release timeline for version 2.5.0 of Milvus. I understand that previous estimates mentioned mid-September and then end of October, and I see that progress is being made, although it seems to be moving slower than initially expected. 

I'm particularly looking forward to version 2.5.0, as I might need the option to upgrade Pulsar to version 3.0.x for a production deployment I am planning. I saw in another [thread](https://github.com/milvus-io/milvus/issues/32184) that making this upgrade a user option in version 2.5.0 was being considered, and I'm highly interested in that feature.

Could you share any information regarding when we might expect this version to be available, or any challenges causing the delays?

Thanks for your continuous efforts in improving Milvus!

**Comment by xiaofan-luan** [comment]
Milvus 2.5 is under the final verification stage and could be released in next week.

I think Milvus 2.4 will also be able to use pulsar 3.x. @LoveEachDay  could you help on answering this quesion?

**Comment by LoveEachDay** [comment]
@villdie18 Are you looking to upgrade an existing Pulsar cluster to version 3.0.x that was installed alongside the Milvus Helm chart, or do you plan to create a new Milvus cluster with a new Pulsar 3.0.x cluster?

**Reply by villdie18** [reply]
@LoveEachDay So for context, I have deployed a Milvus instance in GKE using Helm chart (for tag version 2.4.12). It's currently using Pulsar 2.9.5. I would need to upgrade the Pulsar version to 3.0.x on my existing deployment.
